OpenSim/Simulation/Model/PathSpring.h

```diff
@@ -185,6 +207,11 @@ OpenSim_DECLARE_CONCRETE_OBJECT(PathSpring, Force);
         return values;
     }
 
+protected:
```

Comment:
Separate from other `DECLARE_PROPERTY`s and now `protected`?

Reply:
See above.